docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/fenics/dolfinx/dolfinx:v0.7.0
+FROM ghcr.io/fenics/dolfinx/dolfinx:nightly
```

Comment:
Perhaps change this to stable?

Reply:
Since this will be the entrypoint for the documentation, that is built with nightly, we have to use nightly to be consistent. Of course this is up to debate, do we want to follow stable or main branch dolfinx on main?

Reply:
We can leave it as `nightly` for now. My only concern is that there is a bigger chance that `nigthly` will contain bugs compared to the stable image

Reply:
I propose we change the CI and release system once we actively work on development. 
I think this is something we should discuss with Min and others regarding what is considered best practices.

Reply:
Sounds good 👍